glotaran/builtin/models/kinetic_image/irf.pyi

```diff
@@ -1,50 +1,31 @@
+from __future__ import annotations
+
 from typing import Any
-from typing import List
-from typing import Type
 
-from glotaran.model import model_attribute
-from glotaran.model import model_attribute_typed
+from glotaran.model import model_attribute  # noqa: F401
+from glotaran.model import model_attribute_typed  # noqa: F401
 from glotaran.parameter import Parameter
 
-class IrfMeasured:
-    ...
-
+class IrfMeasured: ...  # noqa: E701
 
 class IrfMultiGaussian:
     @property
-    def center(self) -> List[Parameter]:
-        ...
-
+    def center(self) -> list[Parameter]: ...
     @property
-    def width(self) -> List[Parameter]:
-        ...
-
+    def width(self) -> list[Parameter]: ...
     @property
-    def scale(self) -> List[Parameter]:
-        ...
-
+    def scale(self) -> list[Parameter]: ...
     @property
-    def backsweep_period(self) -> Parameter:
-        ...
-
-    def parameter(self, index: Any):
-        ...
-
-    def calculate(self, index: Any, axis: Any):
-        ...
-
+    def backsweep_period(self) -> Parameter: ...
+    def parameter(self, index: Any): ...
+    def calculate(self, index: Any, axis: Any): ...
 
 class IrfGaussian(IrfMultiGaussian):
     @property
-    def center(self) -> Parameter:
-        ...
-
+    def center(self) -> Parameter: ...
     @property
-    def width(self) -> Parameter:
-        ...
-
+    def width(self) -> Parameter: ...
 
 class Irf:
     @classmethod
-    def add_type(cls, type_name: str, type: Type) -> None:
-        ...
+    def add_type(cls, type_name: str, type: type) -> None: ...
```

Comment:
! type: type

Reply:
Didn't look too closely at the changes since they were autogenerated. Renamed it!